docs/man/man3/settings.3
.TH "settings" 3 "Fri Apr 27 2018" "Variational Analysis" \" -*- nroff -*-
.ad l
.nh
.SH NAME
settings \- 
.SH SYNOPSIS
.br
.PP
.SS "Variables"

.in +1c
.ti -1c
.RI "real(kind=rk8), parameter \fBpstart\fP =200\&.0"
.br
.ti -1c
.RI "real(kind=rk8), parameter \fBpcrit\fP =200\&.0"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnvbudget_column\fP =5"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnvbudget_layer\fP =6"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBntermmax\fP =7"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBntermmaxv\fP =8"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnad\fP =3"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnstx0\fP =10"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnsty0\fP =9"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnvu\fP =10"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBnvs\fP = NVU"
.br
.ti -1c
.RI "real(kind=rk8), parameter \fBpt\fP =90\&.0"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBstart_year\fP =2004"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBstart_month\fP =10"
.br
.ti -1c
.RI "integer(kind=ik4), parameter \fBstart_day\fP = 1"
.br
.in -1c
.SH "Variable Documentation"
.PP 
.SS "integer (kind=ik4), parameter settings::nad =3"

.PP
Definition at line 18 of file settings\&.f90\&.
.PP
.nf
18 INTEGER (KIND=IK4), PARAMETER   :: nad=3                    ! This controls which constraints are used in the analysis
.fi
.SS "integer (kind=ik4), parameter settings::nstx0 =10"

.PP
Definition at line 22 of file settings\&.f90\&.
.PP
.nf
22 INTEGER (KIND=IK4), PARAMETER   :: nstx0=10                 ! The X-size of a grid which covers the network (used in 2D_PUT\&.F90)\&.
.fi
.SS "integer (kind=ik4), parameter settings::nsty0 =9"

.PP
Definition at line 23 of file settings\&.f90\&.
.PP
.nf
23 INTEGER (KIND=IK4), PARAMETER   :: nsty0=9                  ! The Y-size of a grid which covers the network (used in 2D_PUT\&.F90)\&.
.fi
.SS "integer (kind=ik4), parameter settings::ntermmax =7"

.PP
Definition at line 16 of file settings\&.f90\&.
.PP
.nf
16 INTEGER (KIND=IK4), PARAMETER   :: ntermmax=7
.fi
.SS "integer (kind=ik4), parameter settings::ntermmaxv =8"

.PP
Definition at line 17 of file settings\&.f90\&.
.PP
.nf
17 INTEGER (KIND=IK4), PARAMETER   :: ntermmaxv=8
.fi
.SS "integer (kind=ik4), parameter settings::nvbudget_column =5"

.PP
Definition at line 14 of file settings\&.f90\&.
.PP
.nf
14 INTEGER (KIND=IK4), PARAMETER   :: nvbudget_column=5
.fi
.SS "integer (kind=ik4), parameter settings::nvbudget_layer =6"

.PP
Definition at line 15 of file settings\&.f90\&.
.PP
.nf
15 INTEGER (KIND=IK4), PARAMETER   :: nvbudget_layer=6
.fi
.SS "integer (kind=ik4), parameter settings::nvs = NVU"

.PP
Definition at line 26 of file settings\&.f90\&.
.PP
.nf
26 INTEGER (KIND=IK4), PARAMETER   :: nvs= nvu                 ! Number of surface variables\&.
.fi
.SS "integer (kind=ik4), parameter settings::nvu =10"

.PP
Definition at line 25 of file settings\&.f90\&.
.PP
.nf
25 INTEGER (KIND=IK4), PARAMETER   :: nvu=10                   ! Number of upper air variables\&.
.fi
.SS "real (kind=rk8), parameter settings::pcrit =200\&.0"

.PP
Definition at line 12 of file settings\&.f90\&.
.PP
.nf
12 REAL    (KIND=RK8), PARAMETER   :: pcrit=200\&.0              ! 200 hPa OK for the Tropics, no greater than 300 hPa for SGP\&.
.fi
.SS "real (kind=rk8), parameter settings::pstart =200\&.0"

.PP
Definition at line 11 of file settings\&.f90\&.
.PP
.nf
11 REAL    (KIND=RK8), PARAMETER   :: pstart=200\&.0             ! Where to start upper level filtering\&. Not currently used\&.
.fi
.SS "real (kind=rk8), parameter settings::pt =90\&.0"

.PP
Definition at line 28 of file settings\&.f90\&.
.PP
.nf
28 REAL (KIND=RK8), PARAMETER      :: pt=90\&.0                  ! Top of the variational analysis (hPa)
.fi
.SS "integer (kind=ik4), parameter settings::start_day = 1"

.PP
Definition at line 31 of file settings\&.f90\&.
.PP
.nf
31 INTEGER (KIND=IK4), PARAMETER   :: start_day = 1          ! Start day for the variational analysis\&.
.fi
.SS "integer (kind=ik4), parameter settings::start_month =10"

.PP
Definition at line 30 of file settings\&.f90\&.
.PP
.nf
30 INTEGER (KIND=IK4), PARAMETER   :: start_month=10          ! Start month for the variational analysis\&.
.fi
.SS "integer (kind=ik4), parameter settings::start_year =2004"

.PP
Definition at line 29 of file settings\&.f90\&.
.PP
.nf
29 INTEGER (KIND=IK4), PARAMETER   :: start_year=2004          ! Start year for the variational analysis\&.
.fi
.SH "Author"
.PP 
Generated automatically by Doxygen for Variational Analysis from the source code\&.


src/settings.f90
!*************************************************************************************************************************
! This module defines some settings used by the variational analysis code.
!
! Tim Hume.
! 18 August 2006.
!*************************************************************************************************************************

MODULE SETTINGS
USE PORTABLE

REAL    (KIND=RK8), PARAMETER   :: PSTART=200.0             ! Where to start upper level filtering. Not currently used.
REAL    (KIND=RK8), PARAMETER   :: PCRIT=200.0              ! 200 hPa OK for the Tropics, no greater than 300 hPa for SGP.

INTEGER (KIND=IK4), PARAMETER   :: NVBUDGET_COLUMN=5
INTEGER (KIND=IK4), PARAMETER   :: NVBUDGET_LAYER=6
INTEGER (KIND=IK4), PARAMETER   :: NTERMMAX=7
INTEGER (KIND=IK4), PARAMETER   :: NTERMMAXV=8
INTEGER (KIND=IK4), PARAMETER   :: NAD=3                    ! This controls which constraints are used in the analysis
                                                            ! 1 = Mass only
                                                            ! 3 = Mass, heat and water.

INTEGER (KIND=IK4), PARAMETER   :: NSTX0=10                 ! The X-size of a grid which covers the network (used in 2D_PUT.F90).
INTEGER (KIND=IK4), PARAMETER   :: NSTY0=9                  ! The Y-size of a grid which covers the network (used in 2D_PUT.F90).

INTEGER (KIND=IK4), PARAMETER   :: NVU=10                   ! Number of upper air variables.
INTEGER (KIND=IK4), PARAMETER   :: NVS= NVU                 ! Number of surface variables.

REAL (KIND=RK8), PARAMETER      :: PT=90.0                  ! Top of the variational analysis (hPa)
INTEGER (KIND=IK4), PARAMETER   :: START_YEAR=2004          ! Start year for the variational analysis.
INTEGER (KIND=IK4), PARAMETER   :: START_MONTH=10          ! Start month for the variational analysis.
INTEGER (KIND=IK4), PARAMETER   :: START_DAY = 1          ! Start day for the variational analysis.
END MODULE SETTINGS
